app/connectors/propertyLinking/PropertyLinkConnector.scala

```diff
@@ -106,12 +115,34 @@ class PropertyLinkConnector @Inject()(config: ServicesConfig, http: WSHttp)(impl
     http.GET[Option[ClientProperty]](url) recover { case _: NotFoundException => None }
   }
 
-  def getLink(authorisationId: Long)(implicit hc: HeaderCarrier): Future[Option[PropertyLink]] = {
-    http.GET[Option[PropertyLink]](s"$baseUrl/dashboard/assessments/$authorisationId") recover {
+  def getOwnerAssessmentsWithCapacity(submissionId: String, organisationId: Long)(implicit hc: HeaderCarrier): Future[Option[ApiAssessments]] = {
+      http.GET[Option[ApiAssessments]](s"$baseUrl/dashboard/owner/assessments/$submissionId/$organisationId") recover {
+        case _: NotFoundException =>
```

Comment:
this code isnt required.

Reply:
is it the getOwnerAssessmentsWithCapacity method that is not required?

Reply:
the recover on NotFoundException, happens inside a library.

Reply:
removed